openwisp_monitoring/monitoring/migrations/0010_rename_threshold_alert_settings.py

```diff
@@ -0,0 +1,48 @@
+# Generated by Django 3.0.3 on 2020-05-30 19:12
+
+from django.contrib.contenttypes.models import ContentType
+from django.core.exceptions import ObjectDoesNotExist
+from django.core.validators import MaxValueValidator
+from django.db import migrations, models
+
+
+def register_content_type(apps, schema_editor):
+    """Registers AlertSettings if not yet registered"""
+    try:
+        ContentType.objects.get(app_label='monitoring', model='alertsettings')
+    except ObjectDoesNotExist:
+        ct = ContentType(app_label='monitoring', model='alertsettings')
+        ct.save()
+
+
+def unregister_content_type(apps, schema_editor):
+    """Registers AlertSettings if not yet registered"""
+    ct = ContentType.objects.get(app_label='monitoring', model='alertsettings')
+    ct.delete()
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('monitoring', '0009_allow_float_threshold'),
+    ]
+
+    operations = [
+        migrations.AlterModelTable(name='threshold', table='monitoring_alertsettings',),
+        migrations.RenameModel('Threshold', 'AlertSettings'),
+        migrations.AlterField(
+            model_name='alertsettings',
+            name='seconds',
+            field=models.PositiveIntegerField(
+                default=0,
+                help_text='for how long should the alert_settings value be crossed before triggering an alert? The maximum allowed is 604800 seconds (7 days)',
+                validators=[MaxValueValidator(604800)],
+            ),
+        ),
+        migrations.AlterField(
+            model_name='alertsettings',
+            name='value',
+            field=models.FloatField(help_text='alert settings value'),
+        ),
+        migrations.RunPython(register_content_type, unregister_content_type),
```

Comment:
we had issues in the past when doing alters and inserts in the same transaction (one single migration file is executed in a single DB transaction), so move this to a separate migration file please.